src/requirements/data/specializations/en.ts

```diff
@@ -12,4 +12,27 @@ const MATH2940: CollegeOrMajorRequirement = {
   slotNames: ['MATH 1910', 'MATH 1920', 'MATH 2940'],
 };
 
-export default MATH2940;
+const CHEM2080: CollegeOrMajorRequirement = {
+  name: 'Chemistry',
+  description:
+    'CHEM 2090 and CHEM 2080 (can also be substituted by CHEM 2150, PHYS 2214, PHYS 2218, BTRY 3080, ECON 3130, MATH 2930, or MATH 4710)',
```

Comment:
Nit but can there be a period at the end of this description per convention

Reply:
added these, thanks for catching! 

and yes i'll make an item to specialize CHEM reqs and do it in my next pr :)